experiments/saxpy/openmp/ramble.yaml

```diff
@@ -19,22 +19,19 @@ ramble:
     saxpy:
       workloads:
         problem:
-          env_vars:
-            set:
-              OMP_NUM_THREADS: '{n_threads}'
           variables:
             n_ranks: '8'
           experiments:
-            saxpy_{n}_{n_nodes}_{n_ranks}_{n_threads}:
+            saxpy_{n}_{n_nodes}_{n_ranks}_{n_omp_threads_per_task}:
               variables:
                 processes_per_node: ['8', '4']
                 n_nodes: ['1', '2']
```

Comment:
Note: out of `n_nodes`, `processes_per_node`, and `n_ranks`, only 2 of these 3 need to be defined to fully specify what is needed.

Reply:
That makes sense.  Ideas on how we can check that 2 of 3 are defined?  If 3 are defined, we need to check that the math works out.

It looks like n_omp_threads_per_task additionally needs to be defined for OpenMP.  I recommend we replace "task" with "rank" everywhere (they do mean the same thing in this context).